_data/conferences.yml

```diff
@@ -86,6 +86,16 @@
       link: https://www.papercall.io/iosconfsg2025
       deadline: 2024-09-25
 
+- name: Swift DevRoom at FOSDEM 2025
+link: https://swiftlang.github.io/event-fosdem/
```

Comment:
in yml it's important to keep the same indentation

- name:
   link:
   etc: